students/mathewcm/lesson07/html_render.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+
+    def __init__(self, content=None):
+        pass
+
+    def append(self, new_content):
+        pass
+
+    def render(self, out_file):
+        out_file.write("just something as a place holder...")
```

Comment:
@mathewcmartin -- I've been waiting for your completed html_render.py with all the required classes (or at least most of them) implemented, but I don't see those other classes. I can't give a complete mark on this submission. Please do work on this assignment and complete it. Please do ask specific questions in class or at office hours.